### ISSUE TITLE
JAWS No Longer Announcing the Plus Operator when reading MathML with Verbosity of Punctuation=None

### ISSUE THREAD

**Issue by brichwin** [FreedomScientific/VFO-standards-support#538]
### Summary

When reading MathML, JAWS used to announce the equations correctly regardless of the punctuation verbosity setting. Recently JAWS stopped announcing the "plus" operator in equations when the punctuation verbosity is set to none.
  
### Expected result

JAWS should announce the plus operator and other math symbols accurately when reading math content regardless of the punctuation's verbosity setting.

### Actual result

The math content is not being read accurately because the plus operator is not reliably being announced by JAWS.

### Example

1. Go to https://brichwin.pages.iu.edu/tests/silentPlusOperatorMathMLBug_2021-06-04.html 
2. Use quick settings (JAWS Key+V) to set punctuation verbosity to "None"
3. Listen JAWS read the two math equations on the page. Each equation is followed by a text rendering of how JAWS should read them. 
4. Note that the plus operator is not being announced in the equations when the math is read inline with the text.

#### JAWS version and build number
JAWS Version 2021.2105.53 ILM
 
#### Operating System and version
Microsoft Windows 10 Home Version 10.0.19042 Build 19042

#### Browser and version:
Latest Firefox Web Browser Version 89.0 (64 bit)

**Comment by brichwin** [comment]
Hi,

I noticed several other math characters still read incorrectly unless the verbosity setting is set rather high... and what reader really wants that? Any update on this issue? 

This simple equation: ![image](https://user-images.githubusercontent.com/117550/175957932-7ba3f807-4cc5-4e4e-a743-46ed0632cd2d.png) read aloud to one of our students as one plus two divided by three instead of the expected left-parenthesis 1 plus 2 right-parenthesis divided by 3.

The JAWS verbosity setting should not affect how it speaks math symbols. If it's desired not to hear the math, JAWS should just announce the math as some kind of Math object for them to either interact with or skip over. Please don't misrepresent it as your users will only get confused.

**Comment by stevefaulkner** [comment]
@BrettLewisVispero Am unsure about this, really needs to be assessed by someone with knowledge of how JAWS processes/conveys Math.

**Comment by brichwin** [comment]
I've heard via the grape vine that JAWS devs are working on implementing [MathCAT](https://github.com/NSoiffer/MathCAT) as the math handler for JAWS. Using MathCAT and defaulting to output for Blindness, ClearSpeak, and MathCAT's Verbose verbosity setting will solve these issues.

Hopefully the grape vine is correct in this case!
